assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -392,12 +395,12 @@ che.infra.openshift.username=
 che.infra.openshift.password=
 che.infra.openshift.trust_certs=false
 
-che.infra.openshift.che_server_endpoint=http://che-host:${CHE_PORT}/wsmaster/api
-che.infra.openshift.che_server_websocket_endpoint_base=ws://che-host:${CHE_PORT}/wsmaster
+che.infra.openshift.che_server_endpoint=http://${CHE_HOST}:${CHE_PORT}/wsmaster/api
+che.infra.openshift.che_server_default_websocket_endpoint=ws://${CHE_HOST}:${CHE_PORT}/wsmaster/websocket
 
```

Comment:
These properties were added similar to this but in case of the OpenShift we do not have the problem with resolving master endpoints inside a container so we can reuse `che.api`, `che.websocket.endpoint` for OpenShift and rename `che.workspace.che_server*` to docker infra specific.